packages/react/ds-core-form/src/storybook/decorators.tsx

```diff
@@ -0,0 +1,33 @@
+import type React from "react";
+import { FormProvider, useForm } from "react-hook-form";
+
+interface FormDecoratorParams {
+  initialData?: Record<string, unknown>;
+}
+
+export const form = ({ initialData = {} }: FormDecoratorParams = {}) => {
+  return (Story: React.ElementType) => {
+    const FormWrapper: React.ElementType = () => {
+      const methods = useForm({
+        mode: "onChange",
+        defaultValues: initialData,
+      });
+
+      const onSubmit = (data: Record<string, unknown>) => {
+        console.log("[FORM SUBMIT]", data);
+      };
+
```

Comment:
I imagine this is temporary, and later a user will pass their own submit handler, correct?